components/FormElements/formElementsStyle.ts

```diff
@@ -0,0 +1,87 @@
+import styled from 'styled-components'
+import Link from 'next/link'
+
+export const FormStyle = styled.form`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+`
+export const FormHeader = styled.h2`
+  font-size: 2rem;
+  color: ${({ theme }) => theme.text.primary};
+`
+
+export const FieldSetStyle = styled.fieldset`
+  margin: 0 auto;
+  padding: 1rem;
+  width: 100%;
+  box-shadow: 6px 6px 1px 1px ${({ theme }) => theme.bg.tertiary};
+  border: none;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  background: ${({ theme }) => theme.bg.quarternary};
+  align-items: center;
+`
+export const LabelStyle = styled.label`
+  margin-bottom: 0.5rem;
+  text-align: right;
+  width: 100%;
+  color: ${({ theme }) => theme.text.primary};
+  font-size: 1.5rem;
+  display: flex;
```

Comment:
Nice-to-have/nitpick: I see you have these 2 lines a lot. You could abstract it to it's own variable and reuse it where needed, like this:

```javascript
const centeredItems = css`
  display: flex;
  align-items: center;
`

const FirstStyledItem = styled.div`
 background-color: red;
 ${centeredItems}
`

const SecondStyledItem = styled.div`
 background-color: orange;
 ${centeredItems}
`
```

Reply:
Obviously you can and should apply approach where needed

Reply:
this is cool!

Reply:
It is!!!!